site/deploy-site.js

```diff
@@ -0,0 +1,46 @@
+const { execSync } = require('child_process');
+const process = require('process');
+
+const execWithOutput = (cmd) => execSync(cmd, { stdio: 'inherit' });
+
+const DEPLOY_REPOSITORY = 'https://github.com/embark-framework/embark-site';
+const DEPLOY_ORIGIN = 'embark-site';
```

Comment:
Very small thing.  If I understand correctly (maybe not), the term *origin* is more of a convention for the primary remote repository.  The general term is *remote*. See, for example, the OPTIONS section of `man git-push`:

```
OPTIONS
       <repository>
           The "remote" repository that is destination of a push operation. This parameter can be either a
           URL (see the section GIT URLS below) or the name of a remote (see the section REMOTES below).
```

See also: https://github.com/embark-framework/embark/blob/master/scripts/release.js#L21-L22

There could be a good reason to set a default `DEPLOY_REMOTE`, with possibility of override (see link above), and that's the ability to do test releases against a "fake" remote, i.e. a remote that points to another repository on the local filesystem. That's the approach I use (together with verdaccio for the registry aspect) to do local test releases of embark.

Reply:
> The general term is remote. 

This is correct. And semantics are important. Will change this to `DEPLOY_REMOTE`.

> There could be a good reason to set a default DEPLOY_REMOTE, with possibility of override (see link above), and that's the ability to do test releases against a "fake" remote, i.e. a remote that points to another repository on the local filesystem. That's the approach I use (together with verdaccio for the registry aspect) to do local test releases of embark. DEPLOY_REPOSITORY also wouldn't be needed — it's up to the user to make sure the remote points to the correct place, which doesn't seem too unreasonable.

Yes we can make this configurable.